health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/FacilityCatchmentMapping.js

```
@@ -113,7 +113,7 @@ const FacilityCatchmentMapping = () => {
 
 
   useEffect(() => {
-    if (processData && processData.some((instance) => instance.action === "APPROVE_CENSUS_DATA")) {
+    if (processData && processData.some((instance) => instance.action === "FINALIZE_CATCHMENT_MAPPING")) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)